From 2c93eea60434c212770607e23821c33247597ff0 Mon Sep 17 00:00:00 2001
From: Dmitry Rozhkov <dmitry.rozhkov@jollamobile.com>
Date: Mon, 21 Jan 2013 13:04:46 +0200
Subject: [PATCH 1/2] switch docs to default theme

Due to lack of themes in the standard packaged Sphinx switch
to the default theme in order to produce and to package pootle docs.
---
 docs/conf.py |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 042e36f..d468549 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -130,16 +130,16 @@ rst_epilog = """
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'sphinx-bootstrap'
+html_theme = 'default'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = {
-    'nosidebar': True,
-    'link_color': '#489b7',
-    'link_color_hover': '#489b7',
-}
+#html_theme_options = {
+#    'nosidebar': True,
+#    'link_color': '#489b7',
+#    'link_color_hover': '#489b7',
+#}
 
 # Add any paths that contain custom themes here, relative to this directory.
 html_theme_path = ['_themes']
-- 
1.7.10.4

